pkg/instance/start.go

```diff
@@ -117,7 +117,7 @@ func Prepare(ctx context.Context, inst *store.Instance) (*Prepared, error) {
 
 // Start starts the instance.
 // Start calls Prepare by itself, so you do not need to call Prepare manually before calling Start.
-func Start(ctx context.Context, inst *store.Instance, launchHostAgentForeground bool) error {
+func Start(ctx context.Context, inst *store.Instance, limactl string, launchHostAgentForeground bool) error {
```

Comment:
> Needs a comment to explain the limactl arg

Sure, will add one.

> Shouldn't this rather take hostagentCmd []string?

I don't see how this would work. The main purpose of `instance.Start` right now is to assemble the hostagent arguments (after calling `Prepare`), so what else would you pass in besides `["limactl", "hostagent"]`?

Longer term I think we need a way to run the host-agent in-process, in a goroutine, but I'm not quite sure yet how the API should be. This is outside the scope of this PR, which just makes it possible to call `instance.Start`, but not as convenient as it ultimately should be (no dependency on `limactl` at all, ability to hook into event processing).